SimG4Components/src/SimG4OpticalPhysicsList.h

```diff
@@ -0,0 +1,39 @@
+#ifndef SimG4OpticalPhysicsList_h
+#define SimG4OpticalPhysicsList_h 1
+
+// Gaudi
+#include "GaudiKernel/AlgTool.h"
+#include "GaudiKernel/ToolHandle.h"
+
+// FCCSW
```

Comment:
Actually this is the k4FWCore location https://github.com/key4hep/k4FWCore/blob/main/k4Interface/include/k4Interface/ISimG4PhysicsList.h

```suggestion
// k4FWCore
```

But there is also the same file here https://github.com/HEP-FCC/k4SimGeant4/blob/main/SimG4Interface/include/SimG4Interface/ISimG4PhysicsList.h
And it should be dropped in k4SimGeant4.

Reply:
I took as template another physics lists existing in this repo. If refactoring of this and other physics lists tools is needed, maybe it is worth to do it in a separate PR?

Reply:
Yes, probably worth doing a separate PR.